swift/Samples/SampleApp/Sources/AppDelegate.swift

```diff
@@ -9,7 +10,15 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         
         window = UIWindow(frame: UIScreen.main.bounds)
-        window?.rootViewController = ViewController()
+
+        var viewRegistry = ViewRegistry()
+        viewRegistry.register(screenViewControllerType: DemoViewController.self)
+        viewRegistry.register(screenViewControllerType: WelcomeViewController.self)
```

Comment:
Updated. I had not done this, mostly since it maps against the documentation. The extension methods are fairly obvious though, and I do agree making the view controllers fileprivate is nice as well (we might want to consider, outside the scope of it, updating the template to make ScreenViewController's fileprivate by default)